.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup PHP and Composer
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '5.6'
+          php-version: '7.2'
```

Comment:
The php8 compat packages need a later version of PHP. All the linters are running correctly now.